lxd/network_allocations.go

```diff
@@ -155,7 +193,7 @@ func networkAllocationsGet(d *Daemon, r *http.Request) response.Response {
 
 		// Get all the networks, their attached instances, their network forwards and their network load balancers.
 		for _, networkName := range networkNames {
-			if !userHasPermission(entity.NetworkURL(authCheckProjectName, networkName)) {
+			if !canViewNetwork(entity.NetworkURL(authCheckProjectName, networkName)) {
```

Comment:
this should be a separate commit right as is not related to the lease filtering fix?

Reply:
oh wait, you renamed the check for clarity, fine.